### ISSUE TITLE
Drop old connection

### ISSUE THREAD

**Issue by RWOverdijk** [arut/nginx-rtmp-module#367]
Hello,

I'm running into some weird problems. I have two cases where this happens.

**Case No. 1**
Due to a problem with flash, the stream (publisher side) sometimes doesn't drop. What happens, is that the browser is closed but the flash container remains running. This means that the browser you were streaming from is no longer active, but your webcam is still streaming to the server because the flash instance didn't close. When re-opening the page to stream from, I get the "bad name" error.

**Case No. 2**
During streaming (publisher side) the connection sometimes drops. When attempting to reconnect, we get a "bad name" error.

**Solutions tried**
I've tried working with the `drop_idle_publisher` option, to no avail.

**Theoretical solution (workaround)**
What would be good enough to me, is dropping the current stream when a new one comes in with the same name. Is there a way for me to turn off the badName check and just replace the current stream?

**TL;DR**
 Is there a way for me to turn off the badName check and just replace the current stream?

**Comment by UKNickyD** [comment]
I know this is an old post, but I found it by searching for "nginx-rtmp what happens when a second publisher connects to the same stream"... The reason I would like to know is because I would like for my live broadcasters to be able to "switch to adverts", when they want to take a break. I was hoping that if another user connects to the server using the same name and key, the current user would be disconnected and the new user take their place... is this the case or is there any way to implement it?

Thanks in advance and sorry for hijacking an old post.
